Design rules/Diversen.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: page-with-side-nav
+title: API Kennisbank
+---
+
+## 4. Diversen
+
+### DR4.1 Identificatie van een resource zit altijd op het hoogste niveau van de resource
+
+De identificatie van een resource zit  altijd op het hoogste niveau van de resource. Als de identificatie als parameter wordt gebruikt is dat in de vorm en inhoud zoals de identificatie is opgenomen in de resource.
+
+Datum opname : 29-04-2021
+Datum wijziging : 29-04-2021
+
+### DR4.2 Neem voor properties geen waarden op met een speciale betekenis
+
+We nemen geen waarden op met een speciale betekenis die afwijkt van de normale betekenis van het gegeven.
+
+bijvoorbeeld datum "0000-00-00" om aan te geven dat een datum onbekend is
+bijvoorbeeld landcode "0000" om aan te geven dat het land onbekend is
+
+Ratio: Als er informatie beschikbaar is moet die als zondanig onderkend en gemodelleerd worden.
+
+Datum opname : 29-04-2021
+Datum wijziging : 29-04-2021
+
+### DR4.3 De description van een property moet semantisch overeenkomen met de betekenis van het gegeven in een gegevenswoordenboek (infromatiemodel)
+
+We nemen bij een property een description op die semantisch overeenkomt met de beschrijving in het gegevenswoordenboek. Deze kan ingekort, vereenvoudigd, of uitgebreid zijn, maar mag de betekenis van het gegeven niet laten afwijken van de betekenis van het corresponderende gegeven in het gegevenswoordenboek.
+
+De description kan worden weggelaten wanneer evident is dat de gebruikers van de API uit de propertynaam weten wat bedoeld wordt (bijvoorbeeld huisnummer).
+
+_**Ratio:**_ Om de description leesbaar te houden voor developers kan ervoor gekozen worden deze in te korten of
+binnen de context te vereenvoudigen.
+
+Datum opname : 29-04-2021
+Datum wijziging : 29-04-2021
+
+## DR4.4 Als meerdere componenten dezelfde properties bevatten (generalisatie) gebruik dan de 'allOf' om deze samenhang vorm te geven.
```

Comment:
Voeg 1 '#' toe anders staat deze paragraaf op hetzelfde niveau als de paragraaf in regel 6.